docs/index.html

```diff
@@ -6,39 +6,68 @@
 ---
 
 <br />
-				<div id="pgFrontContainer">
-					<!-- <div id="pgFrontMain"> -->
-						<div id="pgFrontFeature">
-							<div id="pgFrontFeatureContent">
-								{% for post in site.posts %}
-								<p class="txtDate"><strong>{{ post.date | date_to_long_string }}</strong></p>
-								<h1 id="txtFrontFeatureHeading">{{ post.title }}</h1>
-								<div>
-								{{ post.excerpt | prepend: "<!--start-->" | split: "<!--start-->" | last }}
-								</div>
-								<p>See full <a href="/documentation/changelog.html#version_{{ post.version}}">changelog for {{ post.version}}</a>
-								{% endfor %}
-								<div id="txtFrontFeatureLink">
-									<br/>
-									<img src="media/img/layout/blt_blu_arrow.png" width="6" height="6" alt="" />
-									<a href="documentation/changelog.html#version_42.1.1" title="Release Notes">Release notes</a>
-									<br/>
-									<img src="media/img/layout/blt_blu_arrow.png" width="6" height="6" alt="" />
-									<a href="download.html" title="Download">Download</a>
-									<br />
-								</div>
-								<div id="txtFrontFeatureLink">
-									<br/>
-									<img src="media/img/layout/blt_blu_arrow.png" width="6" height="6" alt="" />
-									<a href="https://oss.sonatype.org/content/repositories/snapshots/org/postgresql/postgresql/" title="Snapshots">Snapshots can be found here</a>
-									<br />
-								</div>
-								<p>
-									<br/><br/>
-									The PostgreSQL JDBC group would like to thank YourKit for graciously providing licenses to the project
-									<img src="https://www.yourkit.com/images/yklogo.png" />
-								</p>
-							</div> <!-- pgFrontFeatureContent -->
-						</div> <!-- pgFrontFeature -->
-					<!-- </div>  pgFrontMain -->
-				</div> <!-- pgFrontContainer -->
+<div id="pgFrontContainer">
+  <div id="pgFrontMain">
+    <div id="pgFrontFeature">
+      <div id="pgFrontFeatureContent">
+        {% for post in site.posts offset: 0 limit: 3 %}
+        <p class="txtDate"><strong>{{ post.date | date_to_long_string }}</strong></p>
+        <h1 id="txtFrontFeatureHeading">{{ post.title }}</h1>
+        <div>{{ post.excerpt | prepend: "<!--start-->" | split: "<!--start-->" | last }}</div>
+        <br/>
+        <p>See full <a href="documentation/changelog.html#version_{{ post.version}}">changelog for {{ post.version}}</a>
+        <hr />
+        {% endfor %}
+        <!--<div id="txtFrontFeatureLink">
```

Comment:
Is there a reason to keep commented out code?

Reply:
No reason, removed.